pkgs/development/python-modules/scim2-filter-parser/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchPypi, buildPythonPackage, unittestCheckHook, sly }:
+
+buildPythonPackage rec {
+  pname = "scim2-filter-parser";
+  version = "0.4.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-cXHozE4RCSi/24Pl8KGXgdLYsh8x5ZBdkiQ8UGfyz1k=";
+  };
+
+  propagatedBuildInputs = [
+    sly
+  ];
+
+  pythonImportsCheck = [
+    "scim2_filter_parser"
+  ];
+
+  checkInputs = [
+    unittestCheckHook
+  ];
```

Comment:
Are the tests even in the pypi package you're pulling?

```
scim2-filter-parser> unittestCheckPhase
scim2-filter-parser> Executing unittestCheckPhase
scim2-filter-parser> 
scim2-filter-parser> ----------------------------------------------------------------------
scim2-filter-parser> Ran 0 tests in 0.000s
scim2-filter-parser> 
scim2-filter-parser> OK
scim2-filter-parser> Finished executing unittestCheckPhase

Reply:
I believe that there are tests... When I put `tox` in the checkInputs, it ran tests.

Reply:
I don't see any tests in the sdist:

```
❯ atool -x scim2-filter-parser-0.4.0.tar.gz
scim2-filter-parser-0.4.0/LICENSE.txt
scim2-filter-parser-0.4.0/README.rst
scim2-filter-parser-0.4.0/pyproject.toml
scim2-filter-parser-0.4.0/src/scim2_filter_parser/__init__.py
scim2-filter-parser-0.4.0/src/scim2_filter_parser/ast.py
scim2-filter-parser-0.4.0/src/scim2_filter_parser/attr_paths.py
scim2-filter-parser-0.4.0/src/scim2_filter_parser/lexer.py
scim2-filter-parser-0.4.0/src/scim2_filter_parser/parser.py
scim2-filter-parser-0.4.0/src/scim2_filter_parser/queries/__init__.py
scim2-filter-parser-0.4.0/src/scim2_filter_parser/queries/sql.py
scim2-filter-parser-0.4.0/src/scim2_filter_parser/transpilers/__init__.py
scim2-filter-parser-0.4.0/src/scim2_filter_parser/transpilers/django_q_object.py
scim2-filter-parser-0.4.0/src/scim2_filter_parser/transpilers/sql.py
scim2-filter-parser-0.4.0/setup.py
scim2-filter-parser-0.4.0/PKG-INFO
scim2-filter-parser-0.4.0.tar.gz: extracted to `scim2-filter-parser-0.4.0'

Reply:
ok. So should I get rid of the checkInputs entirely?

Reply:
No, instead fetch from a source that has the tests, if there are any.

Reply:
ok. Switched to github tag source